src/platform/graphics/webgpu/webgpu-graphics-device.js

```diff
@@ -51,6 +52,30 @@ class WebgpuGraphicsDevice extends GraphicsDevice {
      */
     computePipeline = new WebgpuComputePipeline(this);
 
+    /**
+     * Buffer used to store arguments for indirect draw calls.
+     *
+     * @type {StorageBuffer|null}
+     * @private
+     */
+    _indirectDrawBuffer = null;
+
+    /**
+     * Number of indirect draw slots allocated.
+     *
+     * @type {number}
+     * @private
+     */
+    indirectDrawBufferCount = 0;
```

Comment:
If this is a private field, should it have a leading underscore? Alternatively, mark it as `@ignore` instead of `@private`?

Reply:
done, thanks